api/konnect/v1alpha1/konnect_extension_types.go

```diff
@@ -103,7 +107,19 @@ type KonnectExtensionControlPlane struct {
 	//
 	// +kubebuilder:validation:Required
 	// +kubebuilder:validation:XValidation:rule="self.type != 'kic'", message="kic type not supported as controlPlaneRef."
-	ControlPlaneRef commonv1alpha1.ControlPlaneRef `json:"controlPlaneRef"`
+	ControlPlaneRef commonv1alpha1.ControlPlaneRef `json:"ref"`
```

Comment:
This could potentially also be renamed to `Ref`, right?

Reply:
Yes, we could rename the go field `Ref`. Honestly, I slightly prefer to keep the longer name as the struct field is not user facing, but mine is not a strong opinion. Do you have some strong opinion here?